gallery/lib/home.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:flutter/material.dart';
+
+import 'demo.dart';
+import 'settings.dart';
+import 'splash.dart';
+
+class HomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Gallery'),
+        leading: IconButton(
+          icon: Icon(Icons.color_lens),
+          onPressed: () {
+            Navigator.of(context).push(
+              MaterialPageRoute(
+                fullscreenDialog: true,
+                builder: (context) => SplashPage(),
+              ),
+            );
+          },
+        ),
+        actions: [
+          IconButton(
+            icon: Icon(Icons.settings),
+            onPressed: () {
+              Navigator.of(context).push(
+                MaterialPageRoute(
+                  fullscreenDialog: true,
+                  builder: (context) => SettingsPage(),
+                ),
+              );
+            },
+          ),
+        ],
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: ListView(
+          children: [
+            Text('Gallery', style: Theme.of(context).textTheme.headline),
+            Container(
+              height: 200.0,
+              child: ListView(
+                scrollDirection: Axis.horizontal,
+                children: [
+                  CarouselCard(title: 'Shrine'),
+                  CarouselCard(title: 'Rally'),
+                  CarouselCard(title: 'Crane'),
+                ],
+              ),
+            ),
+            Text('Categories', style: Theme.of(context).textTheme.headline),
+            CategoryListItem(title: 'Material'),
+            CategoryListItem(title: 'Cupertino'),
+            CategoryListItem(title: 'Reference styles & Media'),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class CarouselCard extends StatelessWidget {
+  const CarouselCard({Key key, this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsetsDirectional.only(
+        top: 8.0,
```

Comment:
drop decimal?

Reply:
Choosing to keep it and make that the style for this project